File: QViz.NET/Periscope.QViz/JSON/TestStepTag.cs
namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Test Case Step Tag Details
	/// </summary>
	public class TestStepTag
	{
		/// <summary>
		/// Identifier of the Test Case Step Tag Details
		/// </summary>
		public string testStepTagId;

		/// <summary>
		/// Tag Object of the Test Case Step
		/// </summary>
		public Tag tag;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public TestStepTag()
		{
			this.testStepTagId = "";
			this.tag = new Tag();
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/TestStepTag.java
package com.periscope.qviz.json;

/**
 * JSON Model for Test Case Step Tag Details
 */
public class TestStepTag {

	/**
	 * Identifier of the Test Case Step Tag Details
	 */
	public String testStepTagId;

	/**
	 * Tag Object of the Test Case Step
	 */
	public Tag tag;

	/**
	 * Default Constructor
	 */
	public TestStepTag() {
		this.testStepTagId = "";
		this.tag = new Tag();
	}

}
